src/adagrams.js

```diff
@@ -1,15 +1,135 @@
+// import { keyOf } from "core-js/core/dict";
+
 export const drawLetters = () => {
   // Implement this method for wave 1
+  const letterPool = {
```

Comment:
Consider moving this outside the function (or wrap it aggressively) so that the function becomes shorter.